docs/dev/search-integration.rst

```diff
@@ -117,6 +117,11 @@ Tags to be ignored:
 
 - ``nav``
 
+Special Sphinx and MkDocs heuristics:
```

Comment:
These are removed for all documentation types, not just mkdocs and sphinx.

Reply:
Yes, I know, the point was more to say that its not really "generic" behavior but behavior that's in the parser because that's how Sphinx does :) I'll try to spell it out. After I wrote it, I didn't find anything special to MkDocs.

This is also in anticipation of having just one generic parser.